helix-term/src/commands.rs

```diff
@@ -1086,18 +1086,27 @@ fn extend_next_long_word_end(cx: &mut Context) {
     extend_word_impl(cx, movement::move_next_long_word_end)
 }
 
-fn will_find_char<F>(cx: &mut Context, search_fn: F, inclusive: bool, extend: bool)
-where
+fn will_find_char<F>(
+    cx: &mut Context,
+    search_fn: F,
+    inclusive: bool,
+    extend: bool,
+    pseudo_pending: Option<String>,
```

Comment:
This can be a `&'static str` and then in the line below you can say `cx.editor.pseudo_pending = Some(key.to_string())`.

Reply:
Might be better to avoid specifying `'static` here as there isn't a hard requirement for it, right? I imagine at some point, to @ath3 's point, we might want to support displaying the remapped keys (which wouldn't be `'static`).